lib/sheldon.rb

```diff
@@ -0,0 +1,51 @@
+require 'fileutils'
+require 'pathname'
+require 'yaml/store'
+require_relative 'sheldon/helpers'
+require_relative 'sheldon/brain'
+
+class Sheldon
+  
+  include Helpers
+
+  def learn(rel_learn_path)
+    abs_learn_path = File.join(Dir.pwd, rel_learn_path)
+    database = load_db 
+  
+    print("Friendly Name For File/Folder: ")
+    recall_cue = STDIN.gets.chomp
+
+    database.transaction do
+      if recall_cue == '' || database[recall_cue]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.